scapy/contrib/automotive/obd/pids_s1AndS2.py

```diff
@@ -0,0 +1,1165 @@
+#! /usr/bin/env python
+
+from scapy.fields import StrFixedLenField
+from scapy.packet import Packet
+
+
+# See https://en.wikipedia.org/wiki/OBD-II_PIDs for further information
+# PID = Parameter IDentification
+
+class Pid00_S1AndS2(Packet):
+    name = "PID_00_PidsSupported"
+    fields_desc = [
+        StrFixedLenField('data', b'', 4)
+    ]
+
+
+class Pid01_S1AndS2(Packet):
+    name = "PID_01_MonitorStatusSinceDtcsCleared"
+    fields_desc = [
+        StrFixedLenField('data', b'', 4)
+    ]
+
+
+class Pid02_S1AndS2(Packet):
+    name = "PID_02_FreezeDtc"
+    fields_desc = [
+        StrFixedLenField('data', b'', 2)
+    ]
+
+
+class Pid03_S1AndS2(Packet):
+    name = "PID_03_FuelSystemStatus"
+    fields_desc = [
+        StrFixedLenField('data', b'', 2)
+    ]
+
+
+class Pid04_S1AndS2(Packet):
```

Comment:
Hmm, I would rather have every class named by its name (below), which sounds clearer. If it is on purpose and has a special meaning, It’s fine

Reply:
The naming convention is explained in the doc:
https://github.com/secdev/scapy/pull/1614/commits/55c77e7be6350868a8b74f26b2d33c61aaec2987#diff-8026137693fcbeb88c642f33c81104cfR1590

Reply:
Ok got it